modules/library/render/src/main/java/org/geotools/renderer/lite/gridcoverage2d/GridCoverageRendererUtilities.java

```diff
@@ -545,6 +545,24 @@ public static List<GridCoverage2D> forceToValidBounds(
         return coverages;
     }
 
+    public static boolean isSameCrs(
+            CoordinateReferenceSystem sourceCrs, CoordinateReferenceSystem targetCrs) {
+        boolean sameCrs = CRS.equalsIgnoreMetadata(sourceCrs, targetCrs);
+        if (!sameCrs) {
+
+            try {
+                sameCrs = !CRS.isTransformationRequired(sourceCrs, targetCrs);
+            } catch (FactoryException e) {
+                // That was an attempt looking for an identity transform.
+                // Let's ignore the exception and return the previous result
+                LOGGER.fine(
+                        "Unable to find MathTransform between sourceCrs and targetCrs, due to"
+                                + e.getLocalizedMessage());
```

Comment:
```suggestion
                LOGGER.log(Level.FINE
                        "Unable to find MathTransform between sourceCrs and targetCrs",  e);
```

Do not eat stack traces